src/algebraic_topology/dold_kan/homotopies.lean

```diff
@@ -0,0 +1,204 @@
+/-
+Copyright (c) 2022 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import algebra.homology.homotopy
+import algebraic_topology.dold_kan.notations
+
+/-!
+
+# Construction of homotopies for the Dold-Kan correspondence
+
+(The general strategy of proof of the Dold-Kan correspondence is explained
+in `equivalence.lean`.)
```

Comment:
This doc string refers to files that do not yet exist in Mathlib. I appreciate you are migrating code from another repo and the doc string is great but I worry about what will happen if something interrupts the migration and we end up with references to missing files.